code/online_challenges/src/project_euler/problem_067/README.md

```diff
@@ -0,0 +1,25 @@
+# Project Euler Problem #067: Maximum path sum II
+
+([Problem Link](https://projecteuler.net/problem=18))
+By starting at the top of the triangle below and moving to adjacent numbers on the row below, the maximum total from top to bottom is 23.
+
+ 3
```

Comment:
You should print this problem instance as a pyramid to make it more readable, and center it.

Reply:
@abdouskamel Can you help me out how to print it as pyramid?